docker-compose.yml

```diff
@@ -0,0 +1,107 @@
+---
+version: '3.7'
+services:
+
+  imsearch:
+    image: imsearch-cpu:latest
+    container_name: ${NAMESPACE}-cpu
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+    - 5000:5000
+    environment:
+    - MONGO_URI=mongodb://mongodb:27017/
+    - REDIS_URI=redis://redis:6379/0
+    - STORAGE_MODE=local
+    depends_on:
+    - redis
+    - mongodb  
+    links:
+    - mongodb:mongodb
+    - redis:redis
+    networks:
+    - internal
+    - web
+
+  mongodb:
+    image: webhippie/mongodb:latest
+    container_name: ${NAMESPACE}-mongodb
+    restart: always
+    ports:
+    - 27017:27017
+    - 28017:28017
+    - 27019:27019
+    volumes:
+    - mongo-data:/var/lib/mongodb/db
+    - mongo-backup:/var/lib/backup
+    networks:
+    - internal
+    environment:
+    - MONGODB_AUTH=${MONGODB_AUTH}
+    - MONGODB_AUTH_DB=${MONGODB_AUTH_DB}
+    - MONGODB_DATABASE=${MONGODB_DATABASE}
+    - MONGODB_DBPATH=${MONGODB_DBPATH}
+    - MONGODB_PASSWORD=${MONGODB_PASSWORD}
+    - MONGODB_REST=${MONGODB_REST}
+    - MONGODB_ROOT_PASSWORD=${MONGODB_ROOT_PASSWORD}
+    - MONGODB_ROOT_USERNAME=${MONGODB_ROOT_USERNAME}
+    - MONGODB_USERNAME=${MONGODB_USERNAME}
+    - MONGODB_VERBOSE=${MONGODB_VERBOSE}
+
+  mongoexpress:
+    restart: always
+    image: mongo-express
+    container_name: ${NAMESPACE}-mongoexpress
+    links:
+    - mongodb:mongo
+    environment:
+    - VCAP_APP_PORT=8090
+    - ME_CONFIG_MONGODB_AUTH_USERNAME=""
+    - ME_CONFIG_MONGODB_AUTH_PASSWORD=""
+    depends_on:
+    - mongodb
+    ports:
+    - 8090:8090
+    networks:
+    - internal
+    - web
+
+  redis:
+    image: redis:5-alpine
+    container_name: ${NAMESPACE}_redis 
+    restart: unless-stopped
+    ports:
+    - "6379:6379"
+    networks:
+    - internal
+
+  imaginary:
```

Comment:
Why this container required?

Reply:
Just removed, it is for resizing images on the fly. I'll use it in my own version

Reply:
Interesting. Let's discuss that how it is used. We can add that in a separate PR.

Reply:
Can you open a new issue to discuss this if you are interested?

Reply:
Yeah definetly, but it is part of the web api I want to create on top of imsearch...

Reply:
okay never mind.